soc/riscv/riscv-privilege/hi3861/scripts/make_burn_bin.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2023 Chen Xingyu <hi@xingrz.me>
```

Comment:
Ditto. If you need to add a custom runner, then you should add it in `scripts/west_commands/runners/`.